main.go

```diff
@@ -67,6 +88,10 @@ func init() {
 }
 
 func main() {
+	configureStatsd()
+	if Statsd != nil {
```

Comment:
`Statsd` can't really be `nil` anymore since `configureStatsd` exits the program if it wasn't able to set it up successfully. I think you can just assume it to be non-nil.